invisible_cities/core/fit_functions.py

```diff
@@ -133,7 +133,7 @@ def power(x, const, pow_):
 
 # ###########################################################
 # Tools
-def fit(func, x, y, seed=(), fit_range=None, **kwargs):
+def fit(func, x, y, seed=(), fit_range=None, full_output=False, **kwargs):
```

Comment:
As discussed offline, remove this argument and enforce it in the call to `scipy.curve_fit` adding the new outputs to the `FitFunction` object.